staging/velero/templates/miniobackend.yaml

```diff
@@ -123,7 +123,7 @@ spec:
       serviceAccountName: minio-operator-sa
       containers:
       - name: minio-operator
-        image: mesosphere/kubeaddons-minio-operator:v0.0.2
+        image: minio/k8s-operator:1.0.1
```

Comment:
can you share the bug that was fixed ? So we can understand why we are changing it :).

Reply:
remember that tempfile problem we paired together on?

so i wrote this to copy the files to not make templfiles and make regular files instead:

https://github.com/s12chung/minio-operator/commit/50afb6c0b43bef4a89153c6087623bc17175b2b5#diff-3254677a7917c6c01f55212f86c57fbfL11

which points to this: https://github.com/s12chung/minio

Reply:
thanks, the explanation would help any reviewer to understand your changes ;)